server/games/game.py

```diff
@@ -391,16 +394,25 @@ def add_game_connection(self, game_connection):
         """
         if game_connection not in self._connections.values():
             return
-        del self._connections[game_connection.player]
-        if game_connection.player:
-            del game_connection.player.game
+
+        player = game_connection.player
+        del self._connections[player]
+
+        if player:
```

Comment:
`if not player` - when that situation will happen?

Reply:
I don't think it can actually.